frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -12,6 +12,15 @@ describe("scenarios > admin > settings", () => {
     signInAsAdmin();
   });
 
+  it("should prompt admin to migrate to the hosted instance", () => {
+    cy.visit("/admin/settings/setup");
+    cy.findByText("Have your server maintained for you.");
+    cy.findByText("Migrate to Metabase Cloud.");
+    cy.findAllByRole("link", { name: "Learn more" })
+      .should("have.attr", "href")
+      .and("include", /migrate/);
```

Comment:
It's likely not going to change, and if it did we still had to support backward compatibility, but wouldn't this be more correct?
```suggestion
      .and("include", "/migrate/");
```